lib/plugins/aws/lib/naming.test.js

```diff
@@ -468,18 +468,6 @@ describe('#naming()', () => {
     });
   });
 
-  describe('#getApiGatewayLogsRoleLogicalId()', () => {
```

Comment:
Do we want tests for the new naming functions?

Reply:
Added